chandra_aca/planets.py

```diff
@@ -134,22 +133,23 @@ def get_planet_angular_sep(
     """
     from agasc import sphere_dist
 
-    if not isinstance(time, CxoTime):
-        time = CxoTime(time)
-
     if observer_position == "earth":
         eci = get_planet_eci(body, time)
         body_ra, body_dec = eci_to_radec(eci)
     elif observer_position == "chandra":
         eci = get_planet_chandra(body, time)
         body_ra, body_dec = eci_to_radec(eci)
     elif observer_position == "chandra-horizons":
-        if time.shape == ():
-            time = CxoTime([time, time + 1000 * u.s])
+        time_secs = convert_time_format(time, "secs")
```

Comment:
`convert_time_format` always returns a numpy array (though it could be a scalar array).